enterprise_gateway/services/sessions/kernelsessionmanager.py

```diff
@@ -66,6 +63,7 @@ def __init__(self, kernel_manager, **kwargs):
         self.kernel_manager = kernel_manager
         self._sessions = dict()
         self._sessionsByUser = dict()
+        self.sessions_to_remove = []
```

Comment:
Now that `delete_sessions` takes a list of kernel_ids, we don't need a member variable for this.

Reply:
Done.